libslide/src/parser/test_utils.rs

```diff
@@ -1,22 +1,29 @@
 #![allow(unused_macros)]
 macro_rules! __parse {
-    ($parser:ident, $program:expr) => {
-        let tokens = scan($program);
+    ($parser:ident, $inout:expr) => {
+        let inout: Vec<&str> = $inout.split(" => ").collect();
+        let pin = inout[0];
+        let pout = if inout.len() > 1 {
+            inout[1].to_owned()
+        } else {
+            pin.to_owned()
+        };
+        let tokens = scan(pin);
```

Comment:
rewrite if into match for consistency. surprised clippy didn't yell at you for this.

Reply:
Actually, clippy yells if I write it in a match.... wants me to use if/else.

Reply:
weird ok.